octa/assetlist.json

```diff
@@ -0,0 +1,27 @@
+{
+  "chain_id": "octa",
+  "assets": [
+    {
+      "description": "The native token of Octa Chain",
+      "denom_units": [
+        {
+          "denom": "uocta",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "octa",
+          "exponent": 6,
+          "aliases": []
+        }
+      ],
+      "base": "uocta",
+      "display": "octa",
+      "symbol": "OCTA",
+      "logo_URIs": {
+        "png": "https://octa-coin-assets.s3.eu-central-1.amazonaws.com/octa-512x512.png",
+        "svg": "https://octa-coin-assets.s3.eu-central-1.amazonaws.com/octa-vector.svg"
```

Comment:
although the png looks like the token logo, the svg looks like a chain logo. A chain logo may be referenced in the chain.json file (see the README for instructions) and titled as `[chain-name]-chain-logo.svg`